### ISSUE TITLE
Support CHECK constraint with procedure call in it

### ISSUE THREAD

**Issue by walkman90** [xnuinside/simple-ddl-parser#76]
Example

```sql
CREATE TABLE foo
(
    entity_id        UUID PRIMARY KEY DEFAULT getId()
    name             VARCHAR(64),
    CONSTRAINT my_constraint  CHECK(my_function(name) IS TRUE)
);
```

Check constraint works well if there is no procedure call in it (my_function).
With procedure call it leads to an empty result `[]`

**Comment by xnuinside** [comment]
@walkman90 thanks for the bug report, I will take care about it at the evening

**Comment by xnuinside** [comment]
I added fix & test for test case https://github.com/xnuinside/simple-ddl-parser/blob/v0.20.0_release/tests/test_simple_ddl_parser.py#L2047. I will release it in version 0.20.0 - I planned to release it on this weekends. I will publish here update, than it will be available on pypi.

**Comment by xnuinside** [comment]
@walkman90 fix was released, but in version 0.19.8 ) - https://pypi.org/project/simple-ddl-parser/ . If you will find anything else - fill free to open the pr
